lib/config/options/index.ts

```diff
@@ -822,7 +822,17 @@ const options: RenovateOptions[] = [
       type: 'string',
       format: 'uri',
     },
-    supportedManagers: ['helm-requirements', 'helmv3', 'helmfile', 'gitlabci'],
+    supportedManagers: [
+      'helm-requirements',
+      'helmv3',
+      'helmfile',
+      'gitlabci',
```

Comment:
Will `gitlabci` manger work if it isn't added in the modules/manager folder?

Reply:
`gitlabci` was already implemented, so it should work.